src/tools/recvfuzz/recvfuzz.cpp

```diff
@@ -547,17 +563,7 @@ void start() {
     }
     QuicAddrSetPort(&sockAddr, 9999);
     // make a server
-    MsQuicRegistration Registration(true);
-    QUIC_SUCCEEDED(Registration.GetInitStatus());
-
-    auto CredConfig = CxPlatGetSelfSignedCert(CXPLAT_SELF_SIGN_CERT_USER, FALSE, NULL);
-
-    MsQuicConfiguration ServerConfiguration(Registration, Alpn, *CredConfig);
-
-    QUIC_SUCCEEDED(ServerConfiguration.GetInitStatus());
-    MsQuicAutoAcceptListener Listener(Registration, ServerConfiguration, MsQuicConnection::NoOpCallback);
-    QUIC_SUCCEEDED(Listener.Start(Alpn, &sockAddr));
-    QUIC_SUCCEEDED(Listener.GetInitStatus());
+    makeServer(&sockAddr);
```

Comment:
Don't put this in a separate function because when the function returns, everything gets cleaned up that was created in it.